config/development.js

```diff
@@ -22,6 +22,7 @@ var config = {
 , debug: true
 , hostname: null
 , port: 4000
+, streamport: 8889
```

Comment:
You need to change it so it has a beginning port and ending port , or a starting port and a number of ports to use after to use.